index.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const DatastoreTrailpack = require('trailpack/datastore');
+const path               = require('path');
+const Wetland            = require('wetland').Wetland;
+
+module.exports = class WetlandTrailpack extends DatastoreTrailpack {
+
+  /**
+   * Validate configured stores
+   */
+  validate() {
+    if (!this.app.config.database || !this.app.config.database.stores) {
+      return this.app.log.logger.warn('No store configured.');
+    }
+  }
+
+  /**
+   * Instantiate wetland with config
+   */
+  configure() {
+    this.app.config.database.orm = 'wetland';
+
+    let wetlandConfig = this.app.config.wetland;
+
+    if (!wetlandConfig) {
```

Comment:
in fact this should be done on the `validate` method witch should check that configuration is present and correct